.gitignore

```diff
@@ -6,3 +6,5 @@ __pycache__
 .idea/
 venv/
 config.sh
+config.py
+.gitignore.swp
```

Comment:
Oh, this shouldn't go in the repo's git ignore, see this [little snippet](https://gist.github.com/mxmeinhold/110d523a9a97ffaa4cec7e506743e6eb#global-ignore) on global ignores, or the man page on the various options. I can also show you my slides on it later.